src/config-inferrer.spec.ts

```diff
@@ -155,5 +155,27 @@ describe('config inferrer', () => {
                 }
             ]
         })
+    ),
+    it("[.NET] nuget",
+        async () => expect({
+            "packages.config": ''
+        },{
+            tasks: [
+                {
+                    init: "nuget install"
+                }
+            ]
+        })
+    ),
+    it("[Ruby] Gemfile",
```

Comment:
Nit: Local style is lowercase:

```suggestion
    it("[ruby] gemfile",
```